src/app/page/wallet-show-page/wallet-show-page.component.html

```diff
@@ -1,3 +1,10 @@
 <h1>
   Wallet Page
 </h1>
+
+<mat-grid-list cols="6" rowHeight="100px">
```

Comment:
don't use rowHeight here, it's better to let the element decide the height and make the height of each row dynamic based on the content.